modules/virt-using-skip-node.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/virt-using-skip-node.adoc
```

Comment:
The assembly is in the node_maintenance directory, not the virtual_machines directory :)